genie-agent/src/main/java/com/netflix/genie/agent/execution/services/impl/grpc/GRpcAgentJobServiceImpl.java

```diff
@@ -117,6 +122,26 @@ public void handshake(
         }
     }
 
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Map<String, String> configure(
```

Comment:
You might want to think about preemptively making this return type some sort of `AgentConfig` object or something so that you can add other things to it if you need to other than just key value pairs?

Reply:
Indeed, that is the proper thing to do here.
I cut a corner since this is relatively easy to change later (server-side only), and didn't want to deal with DTO, converters, etc to just get a map of strings..